### PR TITLE
This is totally gonna work addition feature on deck

### PR DESCRIPTION
**Pull request by ApplebaumIan**
Added addition feature

### CODE REVIEW

File: Sources/WelcomeToGitHub/QuickMaths.swift [hunk 1]
```diff
@@ -16,7 +16,7 @@ struct QuickMaths {
 	///   - rhs: Right hand side of the equation
 	/// - Returns: sum of lhs and rhs as an Int.
 	func add(_ lhs:Int, _ rhs:Int ) -> Int {
-		return 42
+		return lhs + lhs
```
Comment (Agent215) [inline Sources/WelcomeToGitHub/QuickMaths.swift:19]:
```suggestion
		return lhs + rhs
```

**Review by Agent215 — COMMENTED** [review]
yo here

**Review by Agent215 — APPROVED** [review]
looks good

**Review by roburdash — APPROVED** [review]
Looks good

**Pull request merged**